### DISCUSSION TITLE
Inconsistent pronouns in translations

### DISCUSSION THREAD

**Discussion by Andre601** [tuskyapp/Tusky#2948] [General]
Currently going through the german translations, and I'm a bit confused on the main pronouns used in it.

German has multiple variants... For once a more direct one ("Du") and a more formal one ("Sie"). Now, looking at the translations can I find both types being used... Sometimes it's "Du" other times it's "Sie", which makes it more difficult to find out what should be used.

I personally feel like the Du-variant should be utilized here, as it gives a more personal feel to messages rather than a distanced formal one...

Thoughts?

**Comment by charlag** [comment]
I very much would prefer "du"

**Comment by connyduck** [comment]
It should be "du" everywhere
